sdk/eventhub/event-hubs/README.md

```diff
@@ -276,15 +276,21 @@ To control the number of events passed to processEvents, use the options argumen
 You can use `EventHubClient` to work with IotHub as well. This is useful for receiving telemetry data of IotHub from the linked EventHub.
 Most likely the associated connection string will not have send claims. Hence getting HubRuntimeInfo or PartitionRuntimeInfo and receiving events would be the possible operations.
 
-- Please notice that we are awaiting on the [createFromIotHubConnectionString](https://azure.github.io/azure-sdk-for-js/event-hubs/classes/eventhubclient.html#createfromiothubconnectionstring) method to get an instance of the EventHubClient. This is different from other static methods on the client. The method talks to the IotHub endpoint to get a redirect error which contains the EventHub endpoint to talk to. It then constructs the right EventHub connection string based on the information in the redirect error and returns an instance of the EventHubClient that you can play with.
+- Please notice that the connection string needs to contain the `Endpoint` key-value pair
+  in addition to the `SharedAccessKeyName` and `SharedAccessKey` key-value pairs.
+  It can optionally include the name of the IoT Hub as the `EntityPath` value, or the name can be passed separately to the
```

Comment:
The value used from the portal has the `EntityPath` set in it.
There is no other way that the user can get the event hub compatible connection string.
Is there any value talking about `EntityPath` being optional?

Reply:
Now I know how to get the event-hubs compatibile connection string through the portal! When I was getting it through an ARM template, I had to add `EntityPath` manually and through testing found it wasn't required if you specify it in the constructor instead.